homeassistant/components/image_processing/tensorflow.py

```diff
@@ -0,0 +1,336 @@
+"""
+Component that performs TensorFlow classification on images.
+
+For a quick start, pick a pre-trained COCO model from:
+https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/detection_model_zoo.md
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/image_processing.tensorflow/
+"""
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.image_processing import (
+    CONF_CONFIDENCE, CONF_ENTITY_ID, CONF_NAME, CONF_SOURCE, PLATFORM_SCHEMA,
+    ImageProcessingEntity)
+from homeassistant.core import split_entity_id
+from homeassistant.helpers import template
+import homeassistant.helpers.config_validation as cv
+
+REQUIREMENTS = ['numpy==1.15.3', 'pillow==5.2.0', 'protobuf==3.6.1']
+
+_LOGGER = logging.getLogger(__name__)
+
+ATTR_MATCHES = 'matches'
+ATTR_TOTAL_MATCHES = 'total_matches'
+
+CONF_FILE_OUT = 'file_out'
+CONF_MODEL = 'model'
+CONF_GRAPH = 'graph'
+CONF_LABELS = 'labels'
+CONF_MODEL_DIR = 'model_dir'
+CONF_CATEGORIES = 'categories'
+CONF_CATEGORY = 'category'
+CONF_AREA = 'area'
+CONF_TOP = 'top'
+CONF_LEFT = 'left'
+CONF_BOTTOM = 'bottom'
+CONF_RIGHT = 'right'
+
+DEFAULT_MODEL_DIR = "/usr/src/app/tensorflow"
```

Comment:
We should default model dir to our config dir?

Reply:
This is the default hassio config dir.  Is there a helper function available to retrieve the config dir?  If so, I can use that.